From fadd788d26a3a96e9446ad8caee25a0a94d917c4 Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mmfansler@gmail.com>
Date: Tue, 5 Dec 2023 12:25:45 -0500
Subject: [PATCH] fix whitespace

---
 src/Makevars.win | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/Makevars.win b/src/Makevars.win
index 44e9a49..3c7c1b3 100644
--- a/src/Makevars.win
+++ b/src/Makevars.win
@@ -11,11 +11,11 @@ PKG_LIBS =                  \
 all: clean winlibs
 
 winlibs:
- mkdir -p ../inst
- "${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" --vanilla "../tools/winlibs.R" $(VERSION)
- cp -r $(RWINLIB)/share/proj ../inst/
+    mkdir -p ../inst
+    "${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" --vanilla "../tools/winlibs.R" $(VERSION)
+    cp -r $(RWINLIB)/share/proj ../inst/
 
 clean:
-  rm -f $(SHLIB) $(OBJECTS)
+    rm -f $(SHLIB) $(OBJECTS)
 
 .PHONY: all winlibs clean
