From 71bbd311fcef0d566e05e7d106816097a0892695 Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mervin.fansler@bric.ku.dk>
Date: Fri, 10 Jul 2026 13:44:47 +0200
Subject: [PATCH] minimal ucrt libs

---
 src/Makevars.ucrt | 24 +-----------------------
 1 file changed, 1 insertion(+), 23 deletions(-)

diff --git a/src/Makevars.ucrt b/src/Makevars.ucrt
index 6613c50..01f797b 100644
--- a/src/Makevars.ucrt
+++ b/src/Makevars.ucrt
@@ -1,23 +1 @@
-
-ifeq (,$(shell pkg-config --version 2>/dev/null))
-  ## sharpyuv needed, as per proj4 PR by Tomas (march 2023)
-  LIBSHARPYUV = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libsharpyuv.a),-lsharpyuv),)
-  LIBDEFLATE = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libdeflate.a),-ldeflate),)
-  LIBLERC = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/liblerc.a),-llerc),)
-  LIBPSL = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libpsl.a),-lpsl),)
-  LIBBROTLI = $(or $(and $(wildcard $(R_TOOLS_SOFT)/lib/libbrotlidec.a),-lbrotlidec -lbrotlicommon),)
-  PKG_LIBS = -lproj -lsqlite3 -lcurl $(LIBPSL) $(LIBBROTLI) -lbcrypt -ltiff $(LIBLERC) -ljpeg -lrtmp -lssl -lssh2 -lgcrypt -lcrypto -lgdi32 -lz -lzstd -lwebp $(LIBSHARPYUV) $(LIBDEFLATE) -llzma -lgdi32 -lcrypt32 -lidn2 -lunistring -liconv -lgpg-error -lws2_32 -lwldap32 -lwinmm -lstdc++
-else
-  PKG_LIBS = $(shell pkg-config --libs proj)
-  PKG_CPPFLAGS += $(shell pkg-config --cflags proj)
-endif
-
-all: clean winlibs
-
-winlibs:
-	cp -r "$(R_TOOLS_SOFT)/share/proj" ../inst/
-
-clean:
-	rm -f $(SHLIB) $(OBJECTS)
-
-.PHONY: all winlibs clean
+PKG_LIBS = -lproj
